src/api/java/com/ldtteam/structurize/api/util/ItemStackUtils.java

```diff
@@ -47,22 +51,47 @@ private ItemStackUtils()
      */
     public static List<ItemStack> getItemStacksOfTileEntity(final CompoundNBT compound, final World world, final BlockPos pos)
     {
-        final List<ItemStack> items = new ArrayList<>();
         final TileEntity tileEntity = TileEntity.readTileEntity(world.getBlockState(pos), compound);
-        if (tileEntity instanceof LockableTileEntity)
+        if (tileEntity == null)
         {
-            for (int i = 0; i < ((LockableTileEntity) tileEntity).getSizeInventory(); i++)
+            return Collections.emptyList();
+        }
+        tileEntity.setWorldAndPos(world, pos);
```

Comment:
get func has set?

Reply:
It wasn't set, it was crashing without this.

Reply:
seems like a problem somewhere else, should not be a part of this method though 🤷

Reply:
vanilla chests use world n pos to generate the inventory